src/containers/Dashboard.vue

```diff
@@ -198,76 +186,53 @@ export default Vue.extend({
   },
   mounted() {
     this.getInformationFromUser();
+    subscribeToggleableRequirementChoicesChange(() => {
+      // Avoid recomputing requirement before user data is initialized.
+      if (this.user.college !== '') this.recomputeRequirements();
+    });
```

Comment:
To confirm: Does this piece of code mean that we are listening for changes to toggleable requirements, and if there is a change, we recompute requirements, provided that user data is initialized?

Reply:
Yes.